llama_toolchain/agentic_system/api/datatypes.py

```diff
@@ -110,6 +110,12 @@ class Session(BaseModel):
     started_at: datetime
 
 
+@json_schema_type
+class ToolPromptFormat(Enum):
```

Comment:
should this be `CustomToolPromptFormat`? also this one should have a little doc string here as well as in below because this is like the only thing people seem to be asking questions about